src/pages/patterns/disclosures-pattern/index.mdx

```diff
@@ -0,0 +1,779 @@
+and should not act as a screen takeover. Keep all content in a disclosure
+concise and only include information that is necessary. We recommend having a
+width of six columns or less.
+
+#### Disclosures should be user initiated
+
+A disclosure should not take up a considerable amount of the size of the screen.
+Disclosures are meant to be smaller moments that can open on top of different
+areas of a page. They should not seem like a disruption to the users workflow
+and should not act as a screen takeover. Keep all content in a disclosure
+concise and only include information that is necessary. We recommend having a
+width of six columns or less.
+
+#### One disclosure should open at a time
+
+If there are multiple instances on a page where a disclosure is present, only
+one should open at a time to avoid screen clutter and to help the user stay
+focused on the information being disclosed.
+
+<DoDontRow>
+<DoDont
+type="do"
+caption="Do have one disclosure open at a time.">
+
+![Do have one disclosure open at a time.](./images/disclosure-one-at-a-time-do.png)
+
+</DoDont>
+
+<DoDont
+type="dont"
+caption="Do not have multiple disclosures open at the same time.">
+
+![Do not have multiple disclosures open at the same time.](./images/disclosure-one-at-a-time-dont.png)
+
+</DoDont>
+</DoDontRow>
+
+#### Avoid nesting disclosures
+
+Do not nest one disclosure within another disclosure. Nesting disclosures
+creates a stacking effect and could confuse the user of where they should be
+focusing their attention and which disclosure they should be interacting with.
+Using submenus in a context menu that fly out to the sides is an acceptable way
+to disclose additional information in a disclosure.
+
+#### Do not hide critical information within a disclosure
+
+Do not nest one disclosure within another disclosure. Nesting disclosures
+creates a stacking effect and could confuse the user of where they should be
+focusing their attention and which disclosure they should be interacting with.
+Using submenus in a context menu that fly out to the sides is an acceptable way
+to disclose additional information in a disclosure.
+
+#### Use the toggletip component when appropriate
+
+A toggletip follows the disclosure pattern but is also offered as its own
+component with the functionality built in. Use the toggletip component when the
+popover is triggered `on click` with a mouse or by pressing `Enter` or `Space`
+while the trigger element has focus.
+
+### Behaviors
+
+**Opening and closing a disclosure**
+
+To open a disclosure to reveal its content, click the trigger button it is
+related to. To close a disclosure, either click on the trigger button again,
+click anywhere outside of the open disclosure container or click on the close
+`x` icon if it is present within the disclosure.
+
+<Row>
+<Column colLg={8}>
+
+![Example of a disclosure opening and closing.](images/disclosure-opening-closing.gif)
```

Comment:
This example is one I'm not sure about on the disclosure side. I'm curious how we would expect focus to function in this situation and if it's more of a nonmodal dialog type thing

Reply:
Okay, I can take this visual example out or redo it. Does the written content for "Opening and closing a disclosure" make sense though?